pipelines/setup.yml

```diff
@@ -257,3 +257,4 @@ jobs:
                 make -C ./paas-release-ci "${MAKEFILE_ENV_TARGET}" boshrelease-pipelines
                 make -C ./paas-release-ci "${MAKEFILE_ENV_TARGET}" app-deployment-pipelines
                 make -C ./paas-release-ci "${MAKEFILE_ENV_TARGET}" integration-test-pipelines
+                make -C ./paas-release-ci "${MAKEFILE_ENV_TARGET}" plain-pipelines
```

Comment:
This should be in the self-update task and not in the dynamic-pipelines

Reply:
Why so?

The self-update task is for updating the pipeline that's currently running (ie the setup pipeline) before it runs the rest of it. This is the pattern that we're using in many of our pipelines. The plain-pipelines feel like they're another subset of the dynamic pipelines that are created in release-ci and therefore do belong here. (they're also likely to depend on setup having run).

Reply:
After thinking about this, I agree with @alext . 

:+1:

Otherwise, we can simplify this even more, and do everything from the make target  `pipelines`.

Reply:
This is not a strong opinion just how I see our current setup.

This is similar to what we do in create-cloudfoundry. The self-update in create-cloudfoundry updates all pipelines from that repo.
In this case the self update updates the setup pipeline and all other static pipelines and we generate the truly dynamic ones in the dynamic-pipelines task.

Note: I see that even the currently existing pattern is not the best (maybe all pipelines should auto-update only itself).

Reply:
I'm pretty sure they all only update the pipeline being run (the paas-cf ones certainly do). There's an env var that's set to limit the scope of the update-pipelines job to the named pipeline only.

Reply:
@alext I missed that, you're right.

Reply:
> This is not a strong opinion just how I see our current setup.

Neither do, and you are right that it is basically the same.

But I am worried that if we couple these two things (main pipeline `setup` + other) then later we won't be able to customize it further.

Reply:
Let's stay with the setup you created 👍

Reply:
I know about one possible problem, it's going to be harder to iterate on these new plain pipelines as I can't update them from my laptop. I have to commit and run the setup every time.

Reply:
Any reason you can't run `make dev plain-pipelines` from your laptop while iterating on these?

Reply:
@alext I was expecting those to miss some environment variables, but it seems we have everything set.